girder-dandi-archive/tests/rest/test_dandiset_ownership.py

```diff
@@ -0,0 +1,231 @@
+import json
+
+import pytest
+
+from girder.constants import AccessType
+from girder.models.user import User
+from pytest_girder.assertions import assertStatus, assertStatusOk
+
+pytestmark = pytest.mark.plugin("dandi_archive")
+
+
+@pytest.fixture
+def user_2(server):
```

Comment:
Would it be helpful to move some of these fixtures to `conftest.py` in case they're reused in other test files, or is that not likely?

Reply:
b631ba6